test/screenshotter/ss_data.yaml

```diff
@@ -270,7 +270,8 @@ OldFont: |
 OperatorName: |
     \begin{matrix}
         \operatorname g (z) + 5\operatorname{g}z + \operatorname{Gam-ma}(z) \\
-        \operatorname{Gam ma}(z) + \operatorname{\Gamma}(z) + \operatorname{}x
+        \operatorname{Gam ma}(z) + \operatorname{\Gamma}(z) + \operatorname{}x \\
+        \operatorname*{asin} x + \operatorname*{asin}_y x + \operatorname*{asin}\limits_y x
     \end{matrix}
 OpLimits: |
```

Comment:
Was our rendering of `limsup` always wrong?  I'm confused about the image diff since what katex.org currently renders for `\limsup_{x \rightarrow \infty} x \stackrel{?}= \liminf_{x \rightarrow \infty}` is:
<img width="236" alt="Screen Shot 2019-07-13 at 7 03 55 PM" src="https://user-images.githubusercontent.com/1044413/61177342-ff6a0780-a5a0-11e9-93e3-b7614565b109.png">

Reply:
I spotted the following comments in `functions/op.js` which should be removed:
```js
        // TODO(emily): Add a space in the middle of some of these
        // operators, like \limsup
 ``` 
Instead, we already have the following macros in `macros.js` that add `\,` in the middle:
```js
defineMacro("\\limsup", "\\DOTSB\\mathop{\\operatorname{lim\\,sup}}\\limits");
defineMacro("\\liminf", "\\DOTSB\\mathop{\\operatorname{lim\\,inf}}\\limits");
```
If I understand correctly, this PR seems to be disabling underneath limits for `\limsup`, which seems like a bug?

Reply:
In AMS LaTeX, `\limsup`  behaves as does `\sum`. In text style, they both show a sub next to the operator. They show a sub below the operator if in display style or if the operator is followed by `\limits`. Here's an example from quicklatex.com that renders `\( \limsup_x y \: \limsup\limits_x y \: \sum_x^y \: \sum\limits_x^y \:\displaystyle \limsup_x y \: \sum_x^y \)
`
![](https://quicklatex.com/cache3/2b/ql_9eae2b688649ab2d6fcd33f3b8c7092b_l3.png)

 The macro that exists in current KaTeX always shows the sub below a `\limsup` and thus is incorrect when in text style. This PR attempts to get the correct behavior. I'll double check to see that it does and if the screenshots don't show a text style distinction, I'll write a screenshot test case.

Reply:
Ah, I assumed the screenshot was in `\displaystyle`, but if it's in `\textstyle`, that makes more sense! I just checked the definition of `\limsup` and `\liminf` in `amsopn.sty`, and they are clearly just `\operatorname*{lim\,sup}` and `\operatorname*{lim\,inf}`, so I am definitely a fan of the new macros in this PR.